components/director/internal/model/labledef_test.go

```diff
@@ -63,3 +63,51 @@ func TestValidateLabelDef(t *testing.T) {
 	})
 
 }
+
```

Comment:
please rename this file because it contains typo

Reply:
done